core/src/main/scala/filodb.core/query/TransientRow.scala

```diff
@@ -9,6 +10,21 @@ trait MutableRowReader extends RowReader {
   def setBlob(columnNo: Int, base: Array[Byte], offset: Int, length: Int): Unit
 }
 
+final class NaNRowRead(var timestamp: Long) extends RowReader {
```

Comment:
One nitpick: can you rename this to `NanRowReader` to match the naming convention of the other `RowReader`s?

Reply:
Fixed this. Thank you for pointing out this.